src/components/SearchInput.jsx

```diff
@@ -28,6 +33,13 @@ const SearchInput = ({ page, query }) => {
             onChange={changeHandler}
           />
         </form>
+        { searchValue && (
+          <div onClick={ clearHandler } className="absolute right-14 inset-y-0 flex items-center hover:cursor-pointer">
+            <svg width="15" height="15" viewBox="0 0 10 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M0.388258 0.388258C0.452121 0.324233 0.527987 0.273437 0.611512 0.238778C0.695036 0.204119 0.784578 0.186279 0.875008 0.186279C0.965438 0.186279 1.05498 0.204119 1.1385 0.238778C1.22203 0.273437 1.2979 0.324233 1.36176 0.388258L5.00001 4.02788L8.63826 0.388258C8.70218 0.324337 8.77806 0.273632 8.86158 0.239038C8.9451 0.204444 9.03461 0.186639 9.12501 0.186639C9.21541 0.186639 9.30492 0.204444 9.38843 0.239038C9.47195 0.273632 9.54784 0.324337 9.61176 0.388258C9.67568 0.452179 9.72638 0.528064 9.76098 0.61158C9.79557 0.695097 9.81338 0.78461 9.81338 0.875008C9.81338 0.965406 9.79557 1.05492 9.76098 1.13844C9.72638 1.22195 9.67568 1.29784 9.61176 1.36176L5.97213 5.00001L9.61176 8.63826C9.67568 8.70218 9.72638 8.77806 9.76098 8.86158C9.79557 8.9451 9.81338 9.03461 9.81338 9.12501C9.81338 9.21541 9.79557 9.30492 9.76098 9.38843C9.72638 9.47195 9.67568 9.54784 9.61176 9.61176C9.54784 9.67568 9.47195 9.72638 9.38843 9.76098C9.30492 9.79557 9.21541 9.81338 9.12501 9.81338C9.03461 9.81338 8.9451 9.79557 8.86158 9.76098C8.77806 9.72638 8.70218 9.67568 8.63826 9.61176L5.00001 5.97213L1.36176 9.61176C1.29784 9.67568 1.22195 9.72638 1.13844 9.76098C1.05492 9.79557 0.965406 9.81338 0.875008 9.81338C0.78461 9.81338 0.695097 9.79557 0.61158 9.76098C0.528064 9.72638 0.452179 9.67568 0.388258 9.61176C0.324337 9.54784 0.273632 9.47195 0.239038 9.38843C0.204444 9.30492 0.186639 9.21541 0.186639 9.12501C0.186639 9.03461 0.204444 8.9451 0.239038 8.86158C0.273632 8.77806 0.324337 8.70218 0.388258 8.63826L4.02788 5.00001L0.388258 1.36176C0.324233 1.2979 0.273437 1.22203 0.238778 1.1385C0.204119 1.05498 0.186279 0.965438 0.186279 0.875008C0.186279 0.784578 0.204119 0.695036 0.238778 0.611512C0.273437 0.527987 0.324233 0.452121 0.388258 0.388258Z" fill="#11698E"/>
+            </svg>
```

Comment:
Can you move it to assets or a individual component?

Reply:
ready @Katsu08